MicroProfile-Metrics/tck-runner/pom.xml

```diff
@@ -169,10 +151,6 @@
                 </property>
             </activation>
             <properties>
-                <!-- Metrics Dependencies -->
-                <microprofile.metrics.version>2.3</microprofile.metrics.version>
-                <microprofile.metrics.tck.version>2.3</microprofile.metrics.tck.version>
-
                 <!-- Other Test Dependencies -->
                 <cdi.version>2.0.SP1</cdi.version>
```

Comment:
Can remove this too and just update the default property to 2.0.SP1

Reply:
Can we? Community has 2.0 and Enterprise has 2.0.SP1. I'm not sure if that's deliberate? It seemed out of the scope of this issue